src/main/flight/imu.c

```diff
@@ -446,20 +465,30 @@ STATIC_UNIT_TESTED void imuUpdateEulerAngles(void)
     }
 }
 
-static bool imuCanUseAccelerometerForCorrection(void)
+static bool imuCanUseAccelerometerForCorrection(t_fp_vector * grav)
 {
-    int32_t axis;
-    int32_t accMagnitudeSq = 0;
+    float accMagnitudeSq = sq(grav->V.X) + sq(grav->V.Y) + sq(grav->V.Z);
+    const float nearness = ABS(100.0f - 100.0f * accMagnitudeSq / sq(GRAVITY_CMSS));
+    return (nearness > MAX_ACC_SQ_NEARNESS) ? true : false;
```

Comment:
Should be 'false : true'

Because nearness should be small value if 'true'(Close to GRAVITY_CMSS)